### DISCUSSION TITLE
Black Checkmark on Slack Code-Review

### DISCUSSION THREAD

**Discussion by kebab-case-dev** [neuefische/questions#540] [Web] [answered]
Dear Ernst,

could you please comment my code or say anything, what is wrong with my code-review. Because i just get a black checkmark and i am waiting for a green one so i can go on.

[Slack-Link](https://neuefische-students.slack.com/archives/CT7B21ENM/p1678377742619679)

Kind regards,
Bilal

**Comment by Kheirah** [comment, marked as answer]
This is a public forum. You shouldn't link to an internal slack organization :)
I reviewed your PR and requested changes. Check them out and repost your PR for another review when you're done. Mark this thread as resolved or delete it, please.
